### PR TITLE
doc: avoid 'simply'

### PR DESCRIPTION
**Pull request by raboof**
## Description of changes

While the word 'simply' is usually added to encourage readers, it often has the opposite effect and may even appear condescending, especially when the reader runs into trouble trying to apply the suggestions from the documentation. It is almost always an improvement to simply drop the word from the sentence.

(there are more possible improvements like this, we can apply those in separate PRs)

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by drupol — APPROVED** [review]
:+1:

File: nixos/doc/manual/configuration/modularity.section.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Note that both `configuration.nix` and `kde.nix` define the option
 [](#opt-environment.systemPackages). When multiple modules define an
 option, NixOS will try to *merge* the definitions. In the case of
 [](#opt-environment.systemPackages), that's easy: the lists of
-packages can simply be concatenated. The value in `configuration.nix` is
+packages can be concatenated. The value in `configuration.nix` is
```
Comment (rnhmjoj) [inline nixos/doc/manual/configuration/modularity.section.md:40]:
```suggestion
packages are just concatenated. The value in `configuration.nix` is
```

Comment (rnhmjoj) [reply nixos/doc/manual/configuration/modularity.section.md:40]: I think it becomes less readable without "simply" or an equivalent.

File: nixos/doc/manual/development/writing-documentation.chapter.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ pandoc -f markdown_github -t docbook5 docs.md -o my-section.md
 Pandoc can also quickly convert a single `section.xml` to HTML, which is
 helpful when drafting.
 
-Sometimes writing valid DocBook is simply too difficult. In this case,
+Sometimes writing valid DocBook is too difficult. In this case,
```
Comment (rnhmjoj) [inline nixos/doc/manual/development/writing-documentation.chapter.md:53]: I don't see how this cold be interpreted as condescending, it's the opposite, actually.

File: nixos/doc/manual/release-notes/rl-1609.section.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ When upgrading from a previous release, please be aware of the following incompa
 
 Other notable improvements:
 
-- Revamped grsecurity/PaX support. There is now only a single general-purpose distribution kernel and the configuration interface has been streamlined. Desktop users should be able to simply set
+- Revamped grsecurity/PaX support. There is now only a single general-purpose distribution kernel and the configuration interface has been streamlined. Desktop users should be able to set
```
Comment (rnhmjoj) [inline nixos/doc/manual/release-notes/rl-1609.section.md:49]: I don't think it's a good idea to change past release notes retroactively. Same for other changes below.

File: nixos/doc/manual/configuration/modularity.section.md (hunk 1 above)
Comment (raboof) [reply nixos/doc/manual/configuration/modularity.section.md]: I don't know, still seems noisy to me. I tried making it more succinct, WDYT?

File: nixos/doc/manual/development/writing-documentation.chapter.md (hunk 2 above)
Comment (raboof) [reply nixos/doc/manual/development/writing-documentation.chapter.md:53]: I never wanted to suggest it's *always* condescending :) - doesn't seem to add much, either, though.

File: nixos/doc/manual/release-notes/rl-1609.section.md (hunk 3 above)
Comment (raboof) [reply nixos/doc/manual/release-notes/rl-1609.section.md:49]: I can revert them, but I don't see the problem TBH

File: nixos/doc/manual/development/writing-documentation.chapter.md (hunk 2 above)
Comment (rnhmjoj) [reply nixos/doc/manual/development/writing-documentation.chapter.md:53]: Yeah, most of the time is *simply* redundand.

Comment (drupol) [reply nixos/doc/manual/development/writing-documentation.chapter.md:53]: @rnhmjoj If you really want to keep it, you could ~~simply~~ just replace `simply` with `just` !

File: nixos/doc/manual/release-notes/rl-1609.section.md (hunk 3 above)
Comment (rnhmjoj) [reply nixos/doc/manual/release-notes/rl-1609.section.md:49]: It's not really a problem, I think they should be considered of archived.
Besides, practically no one is going to look at them, anyway.

File: nixos/doc/manual/configuration/modularity.section.md (hunk 1 above)
Comment (rnhmjoj) [reply nixos/doc/manual/configuration/modularity.section.md]: Yes, better. The original wording would have made sense if it then showed another example of a type with complex merging rule, but like this is kind of pointless.

**Review by JulienMalka — APPROVED** [review]

File: nixos/doc/manual/development/writing-documentation.chapter.md (hunk 2 above)
Comment (rnhmjoj) [reply nixos/doc/manual/development/writing-documentation.chapter.md:53]: No, I just mean, for something like "a monad is simply a particular monoid in the cathegory of endomorphisms" I see the value in removing simply or rewording, but for most of these I wouldn't have bothered.

**Review by katexochen — APPROVED** [review]

**Review by rnhmjoj — APPROVED** [review]

**Pull request merged**